code/modules/jobs/job_types/antagonists/wizard_apprentice.dm

```diff
@@ -0,0 +1,3 @@
+/datum/job/wizard_apprentice
+	title = "Apprentice" //Turns out that ROLE_WIZARD_APPRENTICE is lower-case, and DB stuff is linked to it.
```

Comment:
Do we actully have any apprentice bans live. Might be worth just popping a script

Reply:
I handle DB stuff very poorly, admittedly. Let me know what change needs to be done and I'll try to do it.